src/pom.xml

```diff
@@ -1026,9 +1026,14 @@
         <version>1.2.17.norce</version>
       </dependency>
       <dependency>
-        <groupId>com.lowagie</groupId>
-        <artifactId>itext</artifactId>
-        <version>2.1.5</version>
+        <groupId>com.github.librepdf</groupId>
+        <artifactId>openpdf</artifactId>
+        <version>1.3.26</version>
+      </dependency>
+      <dependency>
+        <groupId>com.twelvemonkeys.imageio</groupId>
```

Comment:
this dependency does not appear to be used anywhere?  I would expect it to come back in the WMS module, but it doesn't.

it seems openpdf only needs it to _read_ TIFF

Reply:
I left it expecting to need it for another use that didn't turn out. In some ways it serves as a reminder , in other ways it's just noise. 

Would you prefer to see it removed?

Reply:
if not used, I rather see it gone, it could generate noise in the future when there are updates, it could also force unexpected transitive dependency version - though I haven't checked on that

Reply:
Will address.

Reply:
Have removed it.